samples/benchmarks/coremark/src/main.c

```diff
@@ -115,6 +117,19 @@ static int button_init(void)
 	return ret;
 }
 
+static void pre_run_action_perform(void)
+{
+	if (!IS_ENABLED(CONFIG_NRF_ETR)) {
+		return;
+	}
+
+	LOG_INF("Logging is stalled until the logging core finishes the CoreMark benchmark\n");
```

Comment:
`until the application core finishes` - shouldn't be more specific here - it might be not obvious to user that application core is doing logging.

Reply:
Hmm, I wanted to generalize it as we may change the logging core. Perhaps `until this core finishes`? The user gets the domain prefix

Reply:
wouldn't then it say that for every core and "Logging is stalled" only when appcore is busy?

Reply:
I wonder if even we need this as a log. Comment won't be enough?

Reply:
I think the log here is helpful and is not very intrusive (one log per app core benchmark start). When the app core starts executing the benchmark it looks as if the system stalled. Apart from that, users may omit the note from Readme and get confused when logs do not appear when they are running benchmarks on remaining cores.

Reply:
`Logging is blocked for all cores until this core finishes the CoreMark benchmark`